dune-project

```diff
@@ -1,2 +1,104 @@
-(lang dune 1.11)
+(lang dune 3.13)
+
 (name coap)
+
+(generate_opam_files true)
+
+(source
+ (github hyper-systems/ocaml-coap))
+
+(authors "Rizo I. <rizo@odis.io>")
+
+(maintainers "Rizo I. <rizo@odis.io>" "Leo Soares <leojrfs@gmail.com>")
+
+(license ISC)
+
+(documentation https://hyper-systems.github.io/ocaml-coap)
+
+(package
+ (name coap)
+ (synopsis "OCaml CoAP core library")
+ (description
+  "Constrained Application Protocol (RFC 7252) implementation for OCaml.")
+ (depends
+  (ocaml
+   (>= 4.08.0))
+  dune
+  (coap-core
+   (= :version))
+  (coap-server-lwt
+   (= :version))
+  (fmt
+   (>= 0.9.0))
```

Comment:
I don't think we need to constrain the deps so much. I'd leave these unconstrained here and only add bounds when we know there's an incompatibility.

Reply:
ocaml-ci needs it 🥲